packages-tooling/plugin-conventions/src/template-typechecking.ts

```diff
@@ -323,11 +375,15 @@ function tryProcessRepeat(syntax: AttrSyntax, attr: Token.Attribute, node: Defau
     propAccExpr = `${ctx.accessTypeIdentifier}['${identifier}']`;
   } else {
     const rawIterIdentifier = Unparser.unparse(expr.iterable);
-    const [iterIdent, path] = mutateAccessScope(expr.iterable, ctx, member => {
-      const newName = ctx.getIdentifier(member, IdentifierInstruction.AddToOverrides)!;
-      return newName;
-    }, true);
-    propAccExpr = `${ctx.accessTypeIdentifier}${path.map(p => `['${p}']`).join('')}`;
+    const [iterIdent, path] = mutateAccessScope(expr.iterable, ctx, member => ctx.getIdentifier(member, IdentifierInstruction.SkipGeneration) ?? ctx.getIdentifier(member)!, true);
```

Comment:
you probably have a very big screen 😆 
```suggestion
    const [iterIdent, path] = mutateAccessScope(
      expr.iterable,
      ctx,
      member => ctx.getIdentifier(member, IdentifierInstruction.SkipGeneration) ?? ctx.getIdentifier(member)!,
      true
    );
```